### PR TITLE
TDP-4576: Create Ring Verb

### PR DESCRIPTION
**Pull request by CatexB**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4576.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4576.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4576.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4576.s3-website-us-east-1.amazonaws.com/

**Review by n-miles — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4576.s3-website-us-east-1.amazonaws.com/

**Review by muesono-daitan — APPROVED** [review]

File: voice/bxml/verbs/ring.md [hunk 1]
```diff
@@ -30,75 +30,38 @@ This example shows how to use the Ring verb to play ringing audio on a call for
...
 
 ```ruby
-response = Bandwidth::Voice::Response.new()
-ring = Bandwidth::Voice::Ring.new({
-    :duration => 10
-})
-
-response.push(ring)
-puts response.to_bxml()
+// Coming soon
```
Comment (nashley) [inline voice/bxml/verbs/ring.md]:
```suggestion
# Coming soon
```

**Review by nashley — COMMENTED** [review]
Doesn't really matter, but `//` isn't a valid ruby comment afaik

**Review by nashley — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4576.s3-website-us-east-1.amazonaws.com/

**Comment by nashley** [comment]
We can merge this now, right?

**Pull request merged**